src/pages/organize/[orgId]/campaigns/[campId]/calendar/tasks/[taskId]/assignees.tsx

```diff
@@ -12,14 +12,15 @@ import { PageWithLayout } from 'types';
 import { scaffold } from 'utils/next';
 import SingleTaskLayout from 'components/layout/organize/SingleTaskLayout';
 import SmartSearchDialog from 'components/smartSearch/SmartSearchDialog';
+import TaskAssigneesList from 'components/organize/tasks/TaskAssigneesList';
 import { useState } from 'react';
 import getTaskStatus, { TASK_STATUS } from 'utils/getTaskStatus';
 import { ZetkinAssignedTask, ZetkinTask } from 'types/zetkin';
 
 const scaffoldOptions = {
     authLevelRequired: 2,
     localeScope: [
-        'layout.organize', 'misc.breadcrumbs', 'misc.smartSearch', 'pages.assignees', 'misc.tasks',
+        'layout.organize', 'misc.breadcrumbs', 'misc.smartSearch', 'pages.assignees', 'misc.tasks', 'misc.formDialog',
```

Comment:
Great! I've seen this missing before but not gotten around to creating an issue for it. I'm glad you caught it. 😊